src/dom.js

```diff
@@ -0,0 +1,55 @@
+import { format } from 'date-fns';
+
+const form = document.querySelector('.form-location');
+const alertContainer = document.querySelector('.error-container');
+const closeContainer = document.querySelector('.close-btn');
+const sideCol = document.querySelector('.side-col');
+const bgCover = document.querySelector('.bg-cover');
+
+function renderContents() {
```

Comment:
- You should use arrow functions instead of the function keyword throughout your app.`[Optional]`
```suggestion
const renderContents = () => {
```